### DISCUSSION TITLE
New game+, Does anyone know something?

### DISCUSSION THREAD

**Discussion by Rikkuandyuna** [Albeoris/Memoria#455] [Q&A]
Hello, I am trying to create a new game+ after finishing the game but I don't know how to do it, I have tried with Hades Workshop to edit using the field scripts, put a variable in the last text of the game (before seeing the end credits) so that you return to the start of the game.

When I do this, the map changes correctly at first, but the game crashes afterwards. Are there ways to reset all the variables?

**Comment by Tirlititi** [comment]
I have no idea of what should be resetted except the scenario counter. Try adding these lines before going back to Prima Vista's Cargo Room maybe:
```txt
set General_FieldEntrance = 0
set General_ScenarioCounter = 100
```

**Reply by Rikkuandyuna** [reply]
Well it seems that that was the correct way, I put those lines while at the end of the game, and it returned me to the beginning of the game, maintaining my level and the entire team. Do you know by chance how to adjust the amount of HP that regen restores, or the HP that poison removes?

Many thanks for everything!!

**Reply by Tirlititi** [reply]
That's a completly different topic!
It cannot be modified for now, but that's relatively soon on my todo-list.
